<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DungeonResourceDialog</class>
 <widget class="QDialog">
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox">
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Index:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEditWidget" name="indexLineEdit">
        <property name="charWidth">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineEditWidget" name="nameLineEdit">
        <property name="charWidth">
         <number>16</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>File:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="LineEditWidget" name="celFileLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="celFileBrowsePushButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="LineEditWidget" name="widthLineEdit">
        <property name="charWidth">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="frameLabel">
        <property name="text">
         <string>Frame:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="LineEditWidget" name="frameLineEdit">
        <property name="charWidth">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="baseTrnFileLabel">
        <property name="text">
         <string>Base TRN:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="LineEditWidget" name="baseTrnFileLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="baseTrnFileBrowsePushButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="baseTrnFileClearPushButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="uniqueTrnFileLabel">
        <property name="text">
         <string>Unique(TRN):</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QCheckBox" name="uniqueMonCheckBox">
          <property name="toolTip">
           <string>Unique monster</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LineEditWidget" name="uniqueTrnFileLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="uniqueTrnFileBrowsePushButton">
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="uniqueTrnFileClearPushButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Flags:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="preFlagCheckBox">
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <resources/>
 <connections/>
 <buttongroups/>
</ui>
